app/javascript/packs/rating.js

```diff
@@ -16,25 +42,20 @@ window.rating_binder = function() {
     $(this).text('I don\'t want to leave a rating');
     $("#rate").hide("slow");
 
-    $('#rating').raty({
+    $('#rate #rating').raty({
```

Comment:
make it a class if it is being reused on different elements

Reply:
Done